src/tests/ftest/launch.py

```diff
@@ -601,6 +600,8 @@ def __init__(self, name, mode):
         self.slurm_control_node = NodeSet()
         self.slurm_partition_hosts = NodeSet()
         self.slurm_add_partition = False
+        self.core_file_processing = CoreFileProcessing(logger)
```

Comment:
To avoid the import issue with the avocado's settings, let's do this:
```suggestion
        self.core_file_processing = None
```

Reply:
Indeed.  That was the part of the solution I had already figured out.  Where to set it was what I had to do next.